src/GitVersionCore.Tests/VersionCalculation/Strategies/MergeMessageBaseVersionStrategyTests.cs

```diff
@@ -62,19 +67,19 @@ public void TakesVersionFromMergeOfReleaseBranch(string message, bool isMergeCom
             AssertMergeMessage(message + "\n ", expectedVersion, parents);
         }
 
-        [TestCase("Merge branch 'hotfix-0.1.5'", false)]
+        [TestCase("Merge branch 'Feature-0.1.5'", false)]
         [TestCase("Merge branch 'develop' of github.com:Particular/NServiceBus into develop", true)]
         [TestCase("Merge branch '4.0.3'", true)]
         [TestCase("Merge branch 's'", true)]
         [TestCase("Merge tag '10.10.50'", true)]
-        [TestCase("Merge branch 'hotfix-4.6.6' into support-4.6", true)]
-        [TestCase("Merge branch 'hotfix-10.10.50'", true)]
-        [TestCase("Merge branch 'Hotfix-10.10.50'", true)]
-        [TestCase("Merge branch 'Hotfix/10.10.50'", true)]
-        [TestCase("Merge branch 'hotfix-0.1.5'", true)]
-        [TestCase("Merge branch 'hotfix-4.2.2' into support-4.2", true)]
+        [TestCase("Merge branch 'feature-4.6.6' into support-4.6", true)]
+        [TestCase("Merge branch 'Feature-10.10.50'", true)]
+        [TestCase("Merge branch 'Features-10.10.50'", true)]
+        [TestCase("Merge branch 'fix/10.10.50' into hotfix-10.0.0", true)]
+        [TestCase("Merge branch 'feature/0.1.5'", true)]
+        [TestCase("Merge branch 'feature-4.2.2' into support-4.2", true)]
```

Comment:
I assume these test cases broke when the branches were named `hotfix*` and not `feature*`?

Reply:
Yes, every test case scenario which is changed in this PR was failing due to the change.